Makefile

```diff
@@ -24,6 +24,9 @@ production-build: build check-headers-file ## Build the production site and ensu
 non-production-build: ## Build the non-production site, which adds noindex headers to prevent indexing
 	hugo --enableGitInfo
 
+sass-build:
+	scripts/sass.sh build
```

Comment:
minor nit: HUGO_VERSION is set in Makefile and netlify.toml. Is there a better place to set this variable that all scripts/configuration can use?

Reply:
Unfortunately there really isn't a way around that AFAIK. Netlify needs to have that information in the config and the `Makefile` needs it for people to be able to easily build the Docker image. Ideally the `Dockerfile` could extract that information from the Netlify config but I'm not sure there's a way to do that that doesn't add a lot of complexity.